.prettierignore

```diff
@@ -32,3 +32,6 @@ coverage
 
 # forestry files
 .forestry/**/*
+
+# TypeScript example since it uses another eslint config
+examples/using-typescript/**
```

Comment:
This is problematic - it will make lint check not cover using-typescript example. Is it possible to apply eslint settings at root level? If we ignore this in root level, we will not get lint checks for this example on PRs

Reply:
If this is about https://circleci.com/gh/gatsbyjs/gatsby/278730?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link we could make sure to install those plugins in root of monorepo as dev deps